tools/pythonpkg/src/python_import_cache.cpp

```diff
@@ -34,10 +34,10 @@ PyObject *PythonImportCacheItem::AddCache(PythonImportCache &cache, py::object o
 }
 
 void PythonImportCacheItem::LoadModule(const string &name, PythonImportCache &cache) {
-	load_attempted = true;
```

Comment:
I don't agree with this change, since these are modules, if they're not installed at the start of a session, that will not change during the session.
This way we will still attempt to load modules unnecessarily even when we already know they're missing.

I guess monkeypatch might mess with this assumption though..

Reply:
Actually, it's probably fine to leave it as is, because as discussed, the monkeypatch messes with this assumption